### PR TITLE
add it locale

### PR DESCRIPTION
**Pull request by masciugo**

### CODE REVIEW

**Comment by masciugo** [comment]
about the error
```
  1) I18n does not have missing keys
     Failure/Error: expect(missing_keys).to be_empty,
       Missing 11 i18n keys, run `i18n-tasks missing' to show them
     # ./spec/i18n_spec.rb:10:in `block (2 levels) in <top (required)>'
```
with 
```
masciugo@macbookpro ~/dev/administrate $ i18n-tasks missing
Missing translations (7) | i18n-tasks v0.8.7
+--------+-----------------------------------------------+------------------------------------------------------------+
| Locale | Key                                           | Value in other locales or source                           |
+--------+-----------------------------------------------+------------------------------------------------------------+
|  all   | administrate.actions.confirm                  | app/views/administrate/application/_collection.html.erb:69 |
|  all   | administrate.actions.destroy                  | app/views/administrate/application/_collection.html.erb:65 |
|  all   | administrate.actions.edit                     | app/views/administrate/application/_collection.html.erb:59 |
|  all   | administrate.fields.has_many.more             | app/views/fields/has_many/_show.html.erb:30                |
|  all   | administrate.fields.has_many.none             | app/views/fields/has_many/_show.html.erb:39                |
|  all   | administrate.fields.has_one.not_supported     | app/views/fields/has_one/_form.html.erb:22                 |
|  all   | administrate.fields.polymorphic.not_supported | app/views/fields/polymorphic/_form.html.erb:23             |
+--------+-----------------------------------------------+------------------------------------------------------------+
```
I would add the missing keys entrusting the english ones which are missing too..

File: config/locales/administrate.it.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+en:
```
Comment (graysonwright) [inline config/locales/administrate.it.yml]: I think this line needs to be changed to `in:`. This might be what's causing the errors on CircleCI.

**Comment by masciugo** [comment]
ya! thanks

**Comment by graysonwright** [comment]
Grazie!

Can anyone from the community verify this?

**Comment by marcolong** [comment]
Sounds good in italian :)

**Comment by graysonwright** [comment]
Merged this in as 5478293. Thanks!

**Pull request closed without merging**